tests/useAnswersState.test.tsx

```diff
@@ -136,9 +68,9 @@ it('does not trigger render on unmounted component', async () => {
     </AnswersHeadlessContext.Provider>
   );
   act(() => answers.setQuery('resultsWithFilter'));
-  await act( async () => { await answers.executeUniversalQuery(); });
+  await answers.executeUniversalQuery();
```

Comment:
I believe `act()` is to help perform updates so stuff is properly re-render in DOM before you make any assertions. Without it, I'm not too sure just doing `await answers.executeUniversalQuery();` would trigger any updates for the test components to properly test the case of triggering on unmounted component

Reply:
That's fair. I can delete the test